Example/Sources/Data Generation/SampleData.swift

```diff
@@ -40,8 +40,10 @@ final internal class SampleData {
         case Location = 5
         case Url = 6
         case Phone = 7
-        case Custom = 8
-        
+        case Hashtag = 8
+        case Mention = 9
+        case Custom = 10
```

Comment:
Let's make all these cases lowercased, it is not very Swifty 😆 If you could take care of all of them 💯

Reply:
That’s true, It’s not really swift, may I changes all of them in this PR or create another one ? To change others

Reply:
It can all be done in this PR 👍